stdlib/LinearAlgebra/src/symmetric.jl

```diff
@@ -457,6 +457,27 @@ end
 
 *(A::HermOrSym, B::HermOrSym) = A * copyto!(similar(parent(B)), B)
 
+function dot(x::AbstractVector, A::RealHermSymComplexHerm, y::AbstractVector)
+    require_one_based_indexing(x, y)
+    (length(x) == length(y) == size(A, 1)) || throw(DimensionMismatch())
+    Δ = A.uplo == 'U' ? UpperTriangular(A) : LowerTriangular(A)
+    D = Diagonal(A)
```

Comment:
Not used?

Reply:
No, this is a left-over from a previous version.